check-node-version.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+const path = require('path');
+
+const targetVersion = fs.readFileSync(path.resolve(__dirname, '.nvmrc'), { encoding: 'utf8' })
+  // strip trailing and leading whitespace
+  .replace(/\s*$/, '').replace(/^\s*/, '');
+
+if (process.version !== targetVersion) {
+  const separator = '===========================================================';
+  console.warn(
+    separator + '\nUsing the wrong Node version; expected ' + targetVersion + ', got ' + process.version +
```

Comment:
@todo Maybe we make a wiki page about where to find info for updating and add a link to that in this message so people have a next step to take if they have the wrong version?